sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -56,16 +70,33 @@ string json = @"[{""LatinName"":""Animalia"",""Weight"":1.1,""Name"":""Doggo"","
 DogListProperty dog = ModelSerializer.Deserialize<DogListProperty>(BinaryData.FromString(json), options);
 ```
 
-## Using ModelJsonConverter for JsonSerializer
+## Interfacing with protocol methods
```

Comment:
We're making an assumption that developers know what a "protocol method" means to us.   While we use this internally, it's not an industry-standard term and is unlikely to be meaningful to developers using our libraries.

We should define the term, offering a short sentence or three on what a "protocol method" is.  We should also consider a title that doesn't use a non-obvious term.  Maybe something like "Interfacing with low-level methods" or "Working with HTTP methods" or... uh, something.